client/lib/react-pass-to-children/index.js

```diff
@@ -22,7 +22,7 @@ export default function( element, additionalProps ) {
 			return React.cloneElement( child, props );
 		} );
 
-		return React.DOM.div( null, childElements );
+		return <div> { childElements } </div>;
```

Comment:
I think the corresponding tests have to be updated because of the spaces around `childElements`.

Reply:
Fixed with https://github.com/Automattic/wp-calypso/pull/19083/commits/9fa40fdd94d5a17ebde062014c537a3fcdb9799a.

Reply:
thank you so much for that! I had no idea where those spaces were coming from.  Makes sense 👍